src/polychord/interfaces.hpp

```diff
@@ -47,7 +47,7 @@ void run_polychord(
         double (*c_loglikelihood_ptr)(double*,int,double*,int), 
         void (*c_prior_ptr)(double*,double*,int), 
         void (*c_dumper_ptr)(int,int,int,double*,double*,double*,double,double), 
-        void (*c_cluster_ptr)(double*,int*,int), 
+        void (*c_cluster_ptr)(double*,int*,int,int), 
```

Comment:
Good work in catching these. You'll also need to update:
- src/polychord/c_interface.cpp
- src/polychord/interfaces.h
- src/polychord/interfaces.F90